query-node/schema.graphql

```diff
@@ -300,12 +275,17 @@ type Video @entity {
   "Video file metadata"
   mediaMetadata: VideoMediaMetadata
 
-  happenedIn: Block!
+  happenedIn: BigInt!
 
   "Is video featured or not"
   isFeatured: Boolean!
 
   featured: FeaturedVideo @derivedFrom(field: "video")
+
+  # helper properties - redudant info enabling important queries for Joystream use-cases
+
+  is_thumbnail_available: AssetAvailability!
```

Comment:
1. Shouldn't we use `camelCase` here, like in other fields?
2. `is_..._available` suggests a boolean, available or not. Maybe `thumbnailAvailability` is better?

Reply:
1) Yes, that was a mistake.
2) Naming has been changed accordingly.